backend/packages/Upgrade/src/api/services/SegmentService.ts

```diff
@@ -125,32 +143,45 @@ export class SegmentService {
         this.getExperimentSegmentExclusionData(),
       ]);
 
-      const segmentsUsedList = [];
+      const segmentMap = new Map<string, string[]>();
+      segmentsData.forEach((segment) => {
+        if (segment.id === globalExcludeSegment.id) {
+          segmentMap.set(segment.id, []);
```

Comment:
Can't the global segment have sub-segments?

Reply:
Should we have global segment's sub-segments status as "Used"?